src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc

```diff
@@ -59,14 +61,7 @@ CHE_INFRA_KUBERNETES_NAMESPACE_DEFAULT=che-ws-__<username>__
 
 [IMPORTANT]
 ====
-Using the _One namespace per user_ or _One namespace for all workspaces_ strategy with the `common` PVC
-strategy requires persistent volumes with `ReadWriteMany` access mode if more than 1 workspace needs to be running
-concurrently. Because this might be tricky to set up reliably you may want to use a different PVC strategy or, in case
-of _One namespace per user_, to limit the number of concurrently running workspaces per user to 1.
-
-To enforce this limit, set:
+To run more than one workspace at a time when using this strategy together with the `common` PVC strategy, configure persistent volumes to use `ReadWriteMany` access mode.
 
-----
-CHE_LIMITS_USER_WORKSPACES_RUN_COUNT=1
-----
+To limit the number of concurrently running workspaces per user to one, set the `CHE_LIMITS_USER_WORKSPACES_RUN_COUNT` environment variable to `1`.
```

Comment:
I'm not sure if `CHE_LIMITS_USER_WORKSPACE_RUN_COUNT` is mentioned anywhere else, that's why I kept it here.

Reply:
I'm not sure that we should mention env var here at all. It seems to be to low level...
I mean the user should understand where to find the right place to investigate how to deliver such an environment variable according to deployment method -> che-operator, helm chart.
With helm chart - there is no ability to configure this env var, eventually, we should add some configurable value into helm chart.
For che operator - it's customCheProperties field in CheCluster CR https://github.com/eclipse/che-docs/blob/569dec8754b86c28b9b93e82b1f16e0e516b37f4/src/main/pages/che-7/installation-guide/con_che-configmaps-and-their-behavior.adoc#che-installed-using-an-operator

So, I would say here we should have something like
```
To limit the number of concurrently running workspaces per user to one configure:
  - `.global.workspace.number` for helm chart. .... reference to docs that describes how to set value for helm chart.
  - `spec.server.cheCustomProperties.CHE_LIMITS_USER_WORKSPACE_RUN_COUNT` for che operator .... reference to docs that describes how to set value for CheCluster CR
```

Reply:
@metlos, do you agree with @sleshchenko's suggestion? Then we could perhaps merge.

Reply:
:+1:

Reply:
I have no idea about the locations of the mentioned docs though...